crowbar_framework/app/models/node_object.rb

```diff
@@ -1249,6 +1249,19 @@ def poweroff
     bmc_cmd("power off")
   end
 
+  def service(service_name, action)
```

Comment:
While this method name makes sense in the context of this pull request, I'm not convinced the name makes it obvious what this is about outside of this context. Maybe `run_service`?